magpie-aws/pom.xml

```diff
@@ -242,31 +269,35 @@
       <version>1.2.3</version>
       <scope>test</scope>
     </dependency>
-
     <dependency>
-      <groupId>software.aws.mcs</groupId>
-      <artifactId>aws-sigv4-auth-cassandra-java-driver-plugin</artifactId>
-      <version>${aws-sigv4-auth-cassandra-java-driver-plugin.version}</version>
```

Comment:
@kickroot  it is been already there, I have rearranged some dependencies to have better separation of purposes